packages/core/components/FileDetails/index.tsx

```diff
@@ -99,57 +99,55 @@ export default function FileDetails(props: Props) {
             className={classNames(styles.root, styles.expandableTransition, props.className)}
             id={FILE_DETAILS_PANE_ID}
         >
-            <div className={styles.fileDetailsContent}>
```

Comment:
Only change here is I deleted the `<div className={styles.fileDetailsContent}>` element which changed the indentation